cypress/page/address-step.page.ts

```diff
@@ -2,7 +2,7 @@ class AddressStepPage {
     private proceedToCheckoutBtn: string;
 
     constructor() {
-        this.proceedToCheckoutBtn = ".cart_navigation > .button > span";
+        this.proceedToCheckoutBtn = ".cart_navigation > .button[name='processAddress']";
```

Comment:
in this case probably, this can be simplified using descendant notation (whitespace between CSS classes or HTML tags) instead of parenting notation (usage of `>`):
`.cart_navigation .button[name='processAddress']`